src/header/launchbar/Launchbar.scss

```diff
@@ -45,6 +49,10 @@
   width: 32px;
   margin: 0 12px;
 
+  &:first-child {
+    margin-left: 0px;
+  }
+
```

Comment:
Something in the launchbar styles have pushed the buttons to the left of center in between the separators. Compare internal 2020:

<img width="576" alt="image" src="https://user-images.githubusercontent.com/6834224/177284637-7b7fd736-2d59-41be-9c1e-3c027f96264e.png">

with the review deployment:

<img width="524" alt="image" src="https://user-images.githubusercontent.com/6834224/177284747-eb3a32f1-962d-4b5c-8bf2-94ecb01dd4c4.png">